services/minespace-web/src/components/pages/Project/ProjectDescriptionTab.tsx

```diff
@@ -0,0 +1,279 @@
+      "purpose-and-authorization",
+      false
+    );
+    history.push(url);
+  };
+
+  return (
+    <>
+      <Row gutter={[0, 16]}>
+        <Col span={24}>
+          <Row justify="space-between">
+            <Col>
+              <Typography.Title level={4}>Project Description Overview</Typography.Title>
+            </Col>
+            <Col>
+              <Button onClick={handleViewProjectDescriptionClicked} type="primary">
+                {"View Project Description Details"}
+              </Button>
+            </Col>
+          </Row>
+        </Col>
+        <Col span={24}>
+          <Typography.Paragraph>
+            {`Below are the authorization submissions and their status in the project description. Both the Major Mines Office and Ministry of Environments reviews must be completed for this
+           stage to be considered complete.`}
+          </Typography.Paragraph>
+          <Callout
+            style={{ marginTop: 0 }}
+            message={
+              <div className="nod-callout">
+                <h4>{"Submission Failed"}</h4>
+                <p>
+                  {
+                    "One or more of your environment authorizations has not been submitted successfully. Please retry the submission."
+                  }
+                </p>
+              </div>
+            }
+            severity={CALLOUT_SEVERITY.danger}
+          />
+          <Typography.Title level={3}>Submission Progress</Typography.Title>
+          <Typography.Title level={4}>Major Mines Office</Typography.Title>
+          <Typography.Title level={5}>Mines Act</Typography.Title>
+          <CoreTable dataSource={minesActData} columns={minesActColumns} />
+          <Typography.Title level={4}>Ministry of Environment</Typography.Title>
+          <Typography.Title level={5}>Environmental Management Act</Typography.Title>
+          {environmentalManagementActData?.length > 0 && (
+            <Alert
+              message={"Submission Failed Please Retry"}
+              showIcon
+              type="error"
+              description={
+                "One or more of your environment authorization application has not been submitted successfully. Please retry the submission."
+              }
+              action={
+                <Link to={routes.ADD_PROJECT_SUMMARY.dynamicRoute(project.mine_guid)}>
+                  <Button>Retry Failed Submission</Button>
+                </Link>
+              }
+              style={{ marginBottom: "12px" }}
+            />
+          )}
+          <CoreTable dataSource={environmentalManagementActData} columns={otherActColumns} />
+          <br />
+          <Typography.Title level={5}>Water Sustainability Act</Typography.Title>
+          <CoreTable dataSource={waterSustainabilityActData} columns={otherActColumns} />
+          <br />
+          <Typography.Title level={5}>Forestry Act</Typography.Title>
+          <CoreTable dataSource={forestryActData} columns={otherActColumns} />
```

Comment:
It's not a required attribute for CoreTable because it has a default of "key", but it is good to set the rowKey attribute if the data doesn't have a `key` attribute. You may be getting React key errors in the console.